### PR TITLE
feat: add new Chinese translations

### PR DESCRIPTION
**Pull request by picturepan2**
- [ ] feat: add new Chinese translations

### CODE REVIEW

File: packages/frontend/src/translations/zh-hant.global.json [hunk 1]
```diff
@@ -1177,10 +1248,15 @@
...
+        "five": "在第二個彈窗中選擇設備，並點擊連接。",
+        "four": "在首個彈窗中，點擊連接按鈕。",
         "header": "在硬件錢包上安裝 NEAR",
+        "install": "如果你還沒有安裝 NEAR Ledger 應用，請按照",
+        "installLink": "<a href='https://www.ledger.com/ledger-live' target='_blank'>以下步驟操作</a>。",
         "one": "打開 <a href='https://www.ledger.com/ledger-live' target='_blank'>Ledger Live</a>，並安裝固件更新。",
+        "six": "點擊繼續按鈕來完成設置 Ledger。",
         "three": "根據步驟在設備上安裝 <span class='color-black'>NEAR 應用</span>。",
-        "two": "返回至<span class='color-black'>管理器</span> Tab 並搜索 <span class='color-black'>NEAR</span>。"
+        "two": "返回至<span class='color-black'>管理器</span> Tab 並蒐索 <span class='color-black'>NEAR</span>。"
```
Comment (think-in-universe) [inline packages/frontend/src/translations/zh-hant.global.json:1259]:
```suggestion
        "two": "返回至<span class='color-black'>管理器</span> Tab 並搜尋 <span class='color-black'>NEAR</span>。"
```

Comment (think-in-universe) [reply packages/frontend/src/translations/zh-hant.global.json:1259]: it's more native to use "搜尋" for traditional Chinese

**Comment by AmmarHumackicSQA** [comment]
Hey @picturepan2 @think-in-universe, I'm from the Wallet Selector team and was told to reach out to you guys if we can get some help with Chinese translations. You can check [here](https://github.com/near/wallet-selector/blob/dev/packages/core/docs/guides/multilanguage-support.md#adding-new-language) how we add new languages, and if you can support us on it that would be great. Thanks!

**Comment by think-in-universe** [comment]
@AmmarHumackicSQA sure. we could help. will submit a PR this week if I get time.

**Comment by AmmarHumackicSQA** [comment]
> @AmmarHumackicSQA sure. we could help. will submit a PR this week if I get time.

Thank you @think-in-universe that would be great.

**Comment by AmmarHumackicSQA** [comment]
> @AmmarHumackicSQA sure. we could help. will submit a PR this week if I get time.

Hey @think-in-universe, any news on this? When could we expect the PR? Thanks!

**Comment by think-in-universe** [comment]
@AmmarHumackicSQA yeah. I already submit the PR: https://github.com/near/wallet-selector/pull/567

Will do a bit more test and let you know. Let's follow up in the PR.

**Pull request merged**